tests/fix_dns.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+@pytest.fixture(scope="session")
+def psycopg_dns():
```

Comment:
Probably it's ok to have these fixtures in the `fix_psycopg` module.

Reply:
Dropped the fixture, no needed now with `# type: ignore[attr-defined]`.